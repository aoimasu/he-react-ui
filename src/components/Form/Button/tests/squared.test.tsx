import createTestContext from 'react-cosmos-test/enzyme';
import fixture from '../fixtures/squared.fixture';

const { mount, getWrapper } = createTestContext({ fixture });

beforeEach(mount);

test('<Button /> rendered correctly with squared fixture', () => {
  const wrapper = getWrapper();
  expect(wrapper.find('button')).toHaveLength(1);

  expect(wrapper).toMatchSnapshot();
});
