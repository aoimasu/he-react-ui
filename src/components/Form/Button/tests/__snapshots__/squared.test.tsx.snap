// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Button /> rendered correctly with squared fixture 1`] = `
<Button
  buttonProps={Object {}}
  color="teal"
  disabled={false}
  done={false}
  icon=""
  keyline={false}
  link={false}
  newWindow={false}
  onClick={[Function]}
  squared={true}
  submit={false}
  submitting={false}
  withRouter={false}
>
  <div
    className="buttonContainer"
  >
    <InnerButton
      buttonProps={Object {}}
      className="button teal squared"
      newWindow={false}
      onClick={[Function]}
      submit={false}
      withRouter={false}
    >
      <button
        className="button teal squared"
        onClick={[Function]}
        type="button"
      >
        <div
          className="content"
        >
          Button
        </div>
      </button>
    </InnerButton>
  </div>
</Button>
`;
